content/en/real_user_monitoring/browser/troubleshooting.md

```diff
@@ -10,16 +10,17 @@ further_reading:
       text: 'Content Security Policy'
 ---
 
-If you experience unexpected behavior with Datadog Browser RUM, this guide may help resolve issues quickly. Reach out to [Datadog support][1] for further assistance. Regularly update to the latest version of the [RUM Browser SDK][2], as each release contains improvements and fixes.
+If you experience unexpected behavior with Datadog Browser RUM, use this guide to resolve issues quickly. If you continue to have trouble, contact [Datadog support][1] for further assistance. Datadog recommends regularly updating to the latest version of the [RUM Browser SDK][2], as each release contains improvements and fixes.
 
 ## Missing data
 
 If you can't see any RUM data or if data is missing for some users:
 
 | Common causes                                                                                               | Recommended fix                                                                                                                                                                                          |
 | ----------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| Ad blockers prevent the Browser RUM SDK from being downloaded or from sending the data to Datadog.     | Some ad blockers extend their restrictions to performance and marketing tracking tools. [Install the Browser RUM SDK with npm][3] and [forward the collected data through a proxy][4]. |
-| Network rules or VPN prevent the Browser RUM SDK from being downloaded or sending data to Datadog. | Grant access to the endpoints required to download the SDK or to send data. The list of endpoints is available in the [Content Security Policy documentation][5].                                        |
+| Ad blockers prevent the Browser RUM SDK from being downloaded or sending data to Datadog.     | Some ad blockers extend their restrictions to performance and marketing tracking tools. See the [Install the Browser RUM SDK with npm][3] and [forward the collected data through a proxy][4] docs. |
+| Network rules or VPNs prevent the Browser RUM SDK from being downloaded or sending data to Datadog. | Grant access to the endpoints required to download the SDK or to send data. The list of endpoints is available in the [Content Security Policy documentation][5].                                        |
+| Scripts, packages, and clients initialized before the Browser RUM SDK can lead to missed logs, resources, and user actions. | Check when/where the Browser RUM SDK is initialized and consider moving this step earlier in the execution of your application code.                                             |   
```

Comment:
```suggestion
| Scripts, packages, and clients initialized before the Browser RUM SDK can lead to missed logs, resources, and user actions. | Check where in the code the Browser RUM SDK is initialized and consider moving this step earlier in the execution of your application code.  For example, if the Browser RUM SDK is not initialized before ApolloClient initialization, `graphql` requests (shown as XHR resources in RUM Explorer), might not be logged.                                           |   
```
I think you should keep the concrete example provided in the Jira card; it makes both the problem and the fix clearer.